Fix-Website/Templates/content-table.html

```diff
@@ -0,0 +1,9 @@
+﻿<section class="TOC">
```

Comment:
Please use the `.ng` suffix for HTML partials. Why .ng? I have no idea. But it does make it clear that it isn't a complete web page, and only a partial.